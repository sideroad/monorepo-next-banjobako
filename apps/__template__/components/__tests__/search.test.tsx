import React from 'react';
import Search from '../search';
import renderer from 'react-test-renderer';

test('Search to render submit form', () => {
  const component = renderer.create(<Search onSubmit={() => {}} />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
