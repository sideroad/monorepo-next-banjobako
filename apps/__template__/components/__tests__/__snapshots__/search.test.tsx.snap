// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search to render submit form 1`] = `
<form
  className="jsx-1487842596"
  onSubmit={[Function]}
>
  <div
    className="jsx-1487842596 title"
  >
    <h1
      className="jsx-1487842596"
    >
      hello
    </h1>
    <h2
      className="jsx-1487842596"
    >
      search-description
    </h2>
  </div>
  <p
    className="jsx-1487842596"
  >
    <input
      className="jsx-1487842596"
      onChange={[Function]}
      value="octocat"
    />
  </p>
  <p
    className="jsx-1487842596"
  >
    <button
      className="jsx-3752195309"
      onClick={[Function]}
    >
      submit
    </button>
  </p>
</form>
`;
